docs/gcloud-auth.rst

```diff
@@ -58,15 +58,68 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful if your
```

Comment:
I've started doing this and realized it will completely break this diff / PR review.

Is the mental overhead worth it?